llvm/lib/TargetParser/ARMTargetParser.cpp

```diff
@@ -11,9 +11,10 @@
 //
 //===----------------------------------------------------------------------===//
 
-#include "llvm/TargetParser/ARMTargetParser.h"
 #include "llvm/ADT/StringSwitch.h"
+#include "llvm/Support/Format.h"
 #include "llvm/Support/raw_ostream.h"
+#include "llvm/TargetParser/ARMTargetParser.h"
```

Comment:
Again, if you're manually re-ordering these please don't.

If clang-format is doing this, well that's a bit odd but I guess I'll yield to the robot on that one.

Reply:
Will revert.